src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -333,7 +420,11 @@
     /** Options specifically for the case that you are walking a bicycle. */
     public RoutingRequest bikeWalkingOptions;
 
-    /** This is true when a GraphPath is being traversed in reverse for optimization purposes. */
+    /** This is true when a GraphPath is being traversed in reverse for optimization purposes.
+     *
+     * @Deprecated - Not needed in Raptor
+     * */
+    @Deprecated
     public boolean reverseOptimizing = false;
```

Comment:
Info: The Raptor do have a reverse optimization, but it will not be part of the API to turn it on/off - instead we will have profiles were reverse search will be part of some of them.

Reply:
Even though there is reverse optimization in Raptor, we should probably not use the same parameter, as the functionality is a bit different.

Reply:
Agreed, this should not be controlled from the service API.